### ISSUE TITLE
members of a group cannot edit the group, only users in the group

### ISSUE THREAD

**Issue by Gurr1** [cthit/Gamma#402]

**Comment by Portals** [comment]
Resolved in frontend

**Comment by Gurr1** [comment]
Resolving this issue has the side effect of making it possible for a former member of a group to gain privilege escalation, I think.
Will look further into it before resolving or removing it

**Comment by Gurr1** [comment]
Yes, my above comment is correct, I think the best way to resolve this right now is to simply remove the link in the frontend, and look into a way to resolve the problem in a few weeks, when we have more time

**Comment by Portals** [comment]
Since description and function is going to be removed from a group in the `refactor` branch, the only thing worth being able to edit is the avatar for a given group.
